R/fimsframe.R

```diff
@@ -78,11 +90,17 @@ setMethod(
   }
 )
 
+
 setGeneric("m_ages", function(x) standardGeneric("m_ages"))
 setMethod("m_ages", "FIMSFrame", function(x) {
   x@ages
 })
 
+setGeneric("m_comp_lengths", function(x) standardGeneric("m_comp_lengths"))
+setMethod("m_comp_lengths", "FIMSFrame", function(x) {
+  as.numeric(x@comp_lengths)
```

Comment:
substitute "lengths" for "comp_lengths"

Reply:
consistent now that comp is specified for age and length